lib/ansible/modules/notification/slack.py

```diff
@@ -46,6 +46,8 @@
         be locked by your Slack admin and you must request access.  It is there
         that the incoming webhooks can be added.  The key is on the end of the
         URL given to you in that section.
+      - You can also use user or bot tokens, (U(https://api.slack.com/docs/token-types)),
```

Comment:
would you mind add a `Beginning with 2.10, you can...` to indicate the new feature with 2.10

Reply:
Updated :+1: